### DISCUSSION TITLE
ask about a error when the model is running

### DISCUSSION THREAD

**Discussion by Karl131127** [Anura3D/Anura3D_OpenSource#84] [General]
Can anybody help me? I've ran a model with Anura2022, and there was an error like that below.
![image](https://user-images.githubusercontent.com/122240456/226182226-3863f624-8e6d-4866-a969-bb6d744ed09a.png)

**Comment by albuquerqueraniel** [comment]
Hello, Karl131127!

Have you downloaded the Anura code recently? If you didn't, this error may be solved doing so.

I think that this bug was solved, as you can see here:
https://github.com/Anura3D/Anura3D_OpenSource/discussions/72#discussioncomment-5192554

Best regards, 
Raniel

**Reply by Karl131127** [reply]
Thanks for your reply! 
My dear friends. I didn't  download the lastest code, and I'll have a try. If the problem would be solved, I will come here to tell you. 
Thank you again!

**Reply by Karl131127** [reply]
It really works after I compile  latest code.
